site/src/pages/docs/client-decorator.mdx

```diff
@@ -169,6 +169,56 @@ preview because <type://ContentPreviewingClient> is evaluated first for the HTTP
 For <type://RetryingClient>, please check out 
 [`RetryingClient` with logging](/docs/client-retry#retryingclient-with-logging).
 
+## Handling responses
+
+### Modifying response
+
+Modifying the responses can be achieved from within a `ClientBuilder.decorator()` either via using `HttpResponse.aggregate()`, `ClientRequestContext.mutateAdditionalRequestHeaders()` or via a wrapping of the `HttpResponse`, e.g. via `FilteredHttpResponse`:
+
+```java
+import com.linecorp.armeria.common.FilteredHttpResponse;
+import com.linecorp.armeria.common.HttpData;
+import com.linecorp.armeria.common.HttpObject;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.ResponseHeaders;
+
+ClientBuilder cb = Clients.builder(...);
+cb.decorator((delegate, ctx, req) => {
+    HttpResponse response = delegate.execute();
```

Comment:
```suggestion
    HttpResponse response = delegate.execute(ctx, req);
```

Reply:
Another miss. Btw, is the indentation level correct?